### DISCUSSION TITLE
Optional property for Input fields

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#1197] [Q&A] [answered]
The input field component in Sketch has an optional property on the label that looks as below: 

![image](https://user-images.githubusercontent.com/72031986/108217508-973fb800-712b-11eb-8db3-3763bbd26000.png)

I couldn't find the same variant in the Figma component and also couldn't find a prop on the component when using it in my Codesandbox prototype. 

Could you let me know how this can be enabled?

**Comment by richbachman** [comment, marked as answer]
HI @PoojMenon. We never included the optional prop in the component. We have a backlog ticket to do that work. Once its added to the component we'll get it added in Figma as well.

**Reply by PoojMenon** [reply]
Thanks @richbachman. Would you have a timeline for when this might be added? The current project is due to go GA around May/June so would like to know if we'll have it available before then?

**Reply by serifluous** [reply]
Hey @PoojMenon ! We'll have this out in our April 20 release.

As a backup, you can also use the [Text component](https://paste.twilio.design/primitives/text) inside any form label to compose the optional text. Let us know if you need more guidance going with this route.

**Reply by PoojMenon** [reply]
Thats super helpful - thank you!
